calculator_test.go

```diff
@@ -7,18 +7,83 @@ import (
 
 func TestAdd(t *testing.T) {
 	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
-	if want != got {
-		t.Errorf("want %f, got %f", want, got)
+	type testCase struct {
```

Comment:
Good! In fact, we commonly leave the testcase struct anonymous, since it's almost always only used in this one test.

Reply:
Interesting!  I have change to using the anonymous struct style.